tests/test_connection.py

```diff
@@ -44,3 +43,14 @@ def test_Misc(self):
         result[0].uri['nodelist'].pop()
         self.assertTrue(len(result[0].uri['nodelist']) == 0)
         self.assertEqual("Connection()", repr(self.named_conn))
+
+    @defer.inlineCallbacks
+    def test_uri_input(self):
+        test = txmongo.connection.ConnectionPool()
+        yield test.disconnect()
+        test = txmongo.connection.ConnectionPool("mongodb://127.0.0.1/dbname")
+        yield test.disconnect()
+        test = txmongo.connection.ConnectionPool(u"mongodb://127.0.0.1/dbname")
```

Comment:
Was just reading:
The new bytes type is 3.x only. The 2.x bytes built-in is just an alias to the str type. There is no new type called bytes in 2.x; Just a new alias and literal syntax for str.

So bytes, was never really used in py2, it just remapped to str.